mods/ra/maps/singles-in-the-rain/map.yaml

```diff
@@ -0,0 +1,170 @@
+MapFormat: 7
```

Comment:
That's fine for testing, but once this has been demonstrated there's no need to include them.  I think it is a bad idea to ship duplicated maps in our main distribution.

Fort lonestar is set during a thunderstorm and is already configured with rain sound effects.  It will be weird to not have this feature used in it.